### DISCUSSION TITLE
math: Block math in lists isn't rendered as math if the list item also contains inline math

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#17325] [Other Feature Feedback, Questions, & Ideas]
Block math in lists isn't rendered as math if the block math is followed or preceded by inline math:

````markdown
- ```math
  a + b
  ```

  $`c`$
````

> - ```math
>   a + b
>   ```
>
>   $`c`$

> - $`c`$
>
>   ```math
>   a + b
>   ```

Screenshot:

![screenshot](https://user-images.githubusercontent.com/181628/170558853-b9d5cf04-0659-4104-96d8-86526d414829.png)

**Comment by nschloe** [comment]
This is now fixed.

_Edit:_ The bug was reintroduced. :(

**Reply by hsimonfroy** [reply]
Hi,
Just wondering if they were any news from this bug?

I've seen that #40775 and #39545, and your [bug overview](https://github.com/nschloe/markdown-math-acid-test/blob/b0f957daaea46b8de3a0221c7003ffbf9ff64dc1/dollar-backtick.md) also reference it.

At least, are there any ideas how to render properly the intended thing, e.g. with HMTL?
